apps/website/src/content/docs/divider.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Divider
+description: A divider can be used to separate a layout into different sections of content or element.
+propsPath: '@itwin/itwinui-react/esm/core/Divider/Divider.d.ts'
+---
```

Comment:
We should put this component under Utilities in the sidebar. Also we should add a thumbnail in the future. For now lets use a TODO comment.

```suggestion
group: utilities
thumbnail: #TODO
---
```

Reply:
I have added this component under Utilities. Just a quick question, how are thumbnails generated? I noticed some components do have thumbnails while some remain "#TODO".

Reply:
I see, thank you.